DEVELOPERS.md

```diff
@@ -0,0 +1,32 @@
+# Notes for developers
+
+## System requirements
+
+### `just`
+
+```sh
+# macOS
+brew install just
+
+# Linux
+# Install from https://github.com/casey/just/releases
```

Comment:
Manual download because the most recent version in pip is 0.8.138?

Reply:
I didn't know `just` was on PyPI. I think it's probably a different command. Did you `pip install` [this one](https://pypi.org/project/just/), @ccunningham101? If so, then I'll need to make the documentation clearer. There are too many `just`s in the world.

Reply:
Ah yes never mind, that is a different just!